src/pages/event.js

```diff
@@ -231,25 +265,25 @@ export function Event() {
         </div>
         <div className='table-header'>
           <div className='table-title'>Collections <span>({tokens.length})</span></div>
-          <CSVLink
-            filename={`${event.name}.csv`}
-            target="_blank"
-            className="btn"
-            style={{
-              width: 'fit-content',
-              minWidth: 'auto',
-              marginBottom: '0px',
-              marginLeft: 'auto',
-              padding: '12px 32px',
-              display: 'flex',
-              alignItems: 'center',
-              justifyContent: 'center',
-              boxShadow: '0 6px 18px 0 #6534FF4D',
-            }}
-            data={csv_data}
-          >
-            <span>Download CSV</span>
-          </CSVLink>
+          {canDownloadCsv === CanDownloadCsvStatus.Ready || canDownloadCsv === CanDownloadCsvStatus.ReadyWithoutEns ?
+            <CSVLink
+              filename={`${event.name}.csv`}
+              target="_blank"
+              data-tip={`${canDownloadCsv === CanDownloadCsvStatus.ReadyWithoutEns ? 'Please wait if you want the ens names too' : ''}`}
+              className={'btn'}
+              style={csvButtonStyle}
+              data={csv_data}
+            >
+              <span style={{margin: 0}}>{`Download CSV${canDownloadCsv === CanDownloadCsvStatus.ReadyWithoutEns ? ' (without ens)' : ''}`}</span>
+              <ReactTooltip effect={'solid'} />
+            </CSVLink> :
+            <button
+                className={'btn button-disabled'}
+                style={csvButtonStyle}
```

Comment:
Why are we mixing style components with css classes?

Reply:
the classnames are the way of stylizing buttons and the style is specific to this button